src/measure_theory/interval_integral.lean

```diff
@@ -186,12 +186,12 @@ lemma smul [normed_field 𝕜] [normed_space 𝕜 E] {f : α → E} {a b : α} {
   interval_integrable (r • f) μ a b :=
```

Comment:
Is it worth marking this one `simp` too?

Reply:
If I recall correctly, I tried this but it did not accomplish what I had hoped, which is why I added lemmas about multiplication by a constant in #6357. I don't think there's any harm in putting `simp` here, too, but once those other lemmas were added I didn't see the point. 
Is there a specific benefit to adding `simp` here that you had in mind?